tide_core.install

```diff
@@ -1371,3 +1371,17 @@ function tide_core_update_8057() {
     }
   }
 }
+
+/**
+ * Changes the diff modules general_settings.revision_pager_limit to 16.
+ */
+function tide_core_update_8058() {
+  if (\Drupal::moduleHandler()->moduleExists('diff')) {
+    $config = \Drupal::configFactory()
+      ->getEditable('diff.settings');
+    $value = $config->get('general_settings.revision_pager_limit');
```

Comment:
If this value is not set, does this mean `$value` will be falsey? Or are we only setting it if it has some other value?

Reply:
Idea was to check if $value was not empty.